### PR TITLE
added: playlist functionality

### PR DESCRIPTION
**Pull request by rkapoor10**
- default playlists created
 - functionality to add and remove from the existing playlist

### CODE REVIEW

File: src/components/playlists/PlaylistCard.jsx [hunk 1]
```diff
@@ -2,7 +2,13 @@ import { useState } from "react";
 import "./playlistCard.css";
 import { RiPlayList2Fill } from "react-icons/ri";
 import { FaPlay } from "react-icons/fa";
-const PlaylistCard = () => {
+import { useNavigate } from "react-router-dom";
+
+
+
```
Comment (tushartiwari7) [inline src/components/playlists/PlaylistCard.jsx:8]: remove unused spaces.

**Review by tushartiwari7 — APPROVED** [review]
Overall Code looks good and usage of best practices throughout the app.
you can merge this.

**Review by tusharkandpal — APPROVED** [review]
overall the code is clean & well organized.

File: src/components/playlists/PlaylistCard.jsx (hunk 1 above)
Comment (rkapoor10) [reply src/components/playlists/PlaylistCard.jsx:8]: done

**Pull request merged**